lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -283,14 +244,11 @@ sub purge_asset {
         my $tx = $db->begin();
         $db->delete('assets', {filename => $asset});
         $tx->commit;
-        if (-e $asset) {
-            if   (unlink $asset) { $log->debug("CACHE: removed $asset") }
-            else                 { $log->error("CACHE: Could not remove $asset ($!)") }
-        }
-        else { $log->debug("CACHE: requested to remove nonexisting asset $asset") }
+        if (-e $asset) { $log->error("Purging $asset failed: $!") unless unlink $asset }
+        else           { $log->debug("Purging $asset failed because the asset did not exist") }
     };
     if ($@) {
-        $log->error("purge_asset: $@");
+        $log->error("Purging $asset failed: $@");
```

Comment:
This is a bit ambiguous as it is now the same string as in line L247. Could you please use something unique?